model/task.js

```diff
@@ -132,17 +134,27 @@ module.exports = function (app, callback) {
 						return callback(err);
 					}
 					var port = availablePorts.shift();
+					var pa11yOptions = {
+						standard: task.standard,
+						timeout: (task.timeout || 30000),
+						ignore: task.ignore,
+						phantom: {
+							port: port
+						}
+					}
+					if (task.username != '') {
```

Comment:
I think this should be `if (!task.username && !task.password) {`, mostly because otherwise this might fail for existing tests – they will all have a `username` of `undefined` so will try and set it.